java/number-adder/Makefile

```diff
@@ -0,0 +1,13 @@
+all: build
+
+build:
+	mvn package
+	# Copy the produced JAR files to a directory specified by parent Makefile
+	# in the variable used below.
+	cp target/*.jar $(JAVA_BUILD_TARGET_ABSOLUTE)
```

Comment:
Ok. I also added a code that writes an error message if `make build` is called an the required variable is not set.